components/line/actions/send-a-notification/send-a-notification.mjs

```diff
@@ -0,0 +1,80 @@
+import line from "../../line.app.mjs";
+import axios from "axios";
```

Comment:
We like to use the `@pipedream/platform` version of `axios`. See [these docs](https://pipedream.com/docs/pipedream-axios/) for more info.